x-pack/plugins/security_solution/common/utils/risk_score_modules.ts

```diff
@@ -10,7 +10,7 @@ import { RiskScoreEntity, RiskScoreFields } from '../search_strategy';
 import type { Pipeline, Processor } from '../types/risk_scores';
 
 /**
- * * Since 8.5, all the transforms, scripts,
+ * 8.3 and after 8.5, all the transforms, scripts,
```

Comment:
```suggestion
 * All the transforms, scripts,
```
I think the note below about 8.4 is enough to explain, this confused me

Reply:
Or maybe 
```suggestion
 * Aside from 8.4, all the transforms, scripts,
```